pytket/docs/circuit_class.rst

```diff
@@ -338,7 +338,7 @@ condition on a specified set of bit values.)
 
    .. Note:: For adding boxes to a circuit the :py:meth:`Circuit.add_gate` method is sufficient to append any :py:class:`OpType` to a :py:class:`Circuit`.
 
-   .. jupyter-input::
+   ::
```

Comment:
What effect does this have?

Reply:
None really. It just doesn't use the jupyter sphinx package.

It'll still show as a code snippet